accounts/views.py

```diff
@@ -32,4 +32,22 @@ def post(self, request):
             )
             return JsonResponse ({"MESSAGE":"SUCCESS"}, status = 201)
         except KeyError:
-            return JsonResponse ({"MESSAGE":"KEY_ERROR"}, status = 400)
\ No newline at end of file
+            return JsonResponse ({"MESSAGE":"KEY_ERROR"}, status = 400)
+
+class SigninView(View):
+    def get(self, request):
+        data = json.loads(request.body)
+        
+        try:
+            user = Account.objects.get(email=data["email"])
+            if not Account.objects.get(email=data["email"]).exists():
+                return JsonResponse({"MESSAGE":"NOT EXISTED EMAIL"}, status = 400)
+
+            if bcrypt.checkpw(data["password"].encode("utf-8"), user.password.encode("utf-8")):
+                access_token = jwt.encode({"user_id":user.id}, SECRET_KEY, algorithm = 'HS256')
+                return JsonResponse ({"MESSAGE":"SUCCESS", "TOKEN":access_token}, status = 200)
+
+            return JsonResponse({"MESSAGE":"INVALID ERROR"}, status = 400)
```

Comment:
```suggestion
            return JsonResponse({"MESSAGE":"INVALID USER"}, status = 400)
```
위와 같은 이유로 다음과 같이 바꿔주면 좋을 것 같아요

Reply:
넵 수정했습니다-!